Fakebook.Posts/Fakebook.Posts.RestApi/Controllers/PostsController.cs

```diff
@@ -100,7 +105,7 @@ public async Task<IActionResult> PutAsync(int id, Post post)
         [ProducesResponseType(StatusCodes.Status201Created)]
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
-        public async Task<IActionResult> PostAsync(NewPostDto postModel)
+        public async Task<IActionResult> PostAsync([Required]NewPostDto postModel)
```

Comment:
i think this is superfluous, but see what happens if you send in the body `null`, or nothing (which i don't think is valid JSON) with and without this attribute.
if it's superfluous, you could mark the code smells as false positive

Reply:
I sent in the body as actual null {Content: null} and also just an empty object and both were giving me bad request with or without [Required] and that's what I want so Im going to call that code smell a false positive and be done with it.